### DISCUSSION TITLE
[gestalt] Link missing underline prop

### DISCUSSION THREAD

**Discussion by andrewlutz** [DefinitelyTyped/DefinitelyTyped#63705] [Issues with a @types package]
`Link`  component in Gestalt is missing the `underline` prop https://gestalt.pinterest.systems/web/link

[a comment by an automated account is not shown]

**Comment by andrewlutz** [comment]
Added in PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63706

**Comment by SGaofei** [comment]
您好！您的邮件我已收到，谢谢啦！
